### ISSUE TITLE
Add conformance test for proxied and redirected binary content

### ISSUE THREAD

**Issue by escowles** [cavendish-ldp/cavendish#14]
Verify support for external content using message/external with URL and local-file access-types:

* message/external-body spec: https://www.w3.org/Protocols/rfc1341/7_3_Message.html
* access-type=url spec: https://tools.ietf.org/rfc/rfc2017.txt

**Comment by barmintor** [comment]
This part of the spec has changed a bit, but I spoke with @awoods about it today.

**Comment by barmintor** [comment]
@escowles is there a write-up of what the MODE alignment decision was re: proxied content?

**Comment by escowles** [comment]
@barmintor No, I don't think there's really been a firm agreement on that amongst the spec editors, and so I'm not exactly sure what the folks working on the MODE alignment sprints have done (or if they are waiting on us).

But my understanding of the current thinking is in these two tickets:
* https://github.com/fcrepo/fcrepo-specification/issues/236 — require support for Content-type=message/external-body; access-type=URL (with a note that an impl may copy, cache, ingest, etc. if it wants to)
* https://github.com/fcrepo/fcrepo-specification/issues/237 — ingest-by-reference using the Location header to specify the remote content location.
